New jooby project with jetty won't compile: `package io.jooby.netty does not exist`

Hello,

I got stuck following the getting started instructions.

I ran the following:
```
jooby set -w ~/dev/JoobyWs/
jooby
jooby> create demo --server jetty
cd dev/JoobyWs
mvn jooby:run
```
And got the following compilation error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project demo: Compilation failure: Compilation failure: 
[ERROR] /home/axel/dev/JoobyWs/demo/src/main/java/app/App.java:[4,22] package io.jooby.netty does not exist
[ERROR] /home/axel/dev/JoobyWs/demo/src/main/java/app/App.java:[9,17] cannot find symbol
[ERROR]   symbol:   class NettyServer
[ERROR]   location: class app.App
```
I am surprised Netty would show up at all after using `--server jetty`. Is there a bug or am I doing something obviously wrong?

Thanks

--- a/modules/jooby-cli/src/main/java/io/jooby/cli/CreateCmd.java
+++ b/modules/jooby-cli/src/main/java/io/jooby/cli/CreateCmd.java
@@ -156,6 +156,9 @@ public class CreateCmd extends Cmd {
       finalArtifactId = name + "-" + version + (stork ? ".zip" : ".jar");
     }
 
+    String serverPackageName = serverPackageName(server);
+    String serverClassName = serverClassName(server);
+
     Map<String, Object> model = new HashMap<>();
     Map<String, String> dependencyMap = ctx.getDependencyMap();
     model.putAll(dependencyMap);
@@ -166,6 +169,8 @@ public class CreateCmd extends Cmd {
     model.put("version", version);
     model.put("joobyVersion", ctx.getVersion());
     model.put("server", server);
+    model.put("serverClassName", serverClassName);
+    model.put("serverPackageName", serverPackageName);
     model.put("kotlin", kotlin);
     model.put("dependencies", dependencies(dependencyMap, server, kotlin));
     model.put("testDependencies", testDependencies(dependencyMap, kotlin));
@@ -273,6 +278,32 @@ public class CreateCmd extends Cmd {
       default:
         throw new IllegalArgumentException("Unknown server option: " + value);
     }
+  }
+
+  private String serverPackageName(String server) {
+	  switch(server) {
+	  case "jetty":
+		  return "jetty";
+	  case "Netty":
+		  return "netty";
+	  case "utow":
+		  return "undertow";
+      default:
+        throw new IllegalArgumentException("Unknown server value: " + server);
+	  }
+  }
+  
+  private String serverClassName(String server) {
+	  switch(server) {
+	  case "jetty":
+		  return "JettyServer";
+	  case "Netty":
+		  return "NettyServer";
+	  case "utow":
+		  return "UndertowServer";
+      default:
+        throw new IllegalArgumentException("Unknown server value: " + server);
+	  }
   }
 
   private void stork(CliContext ctx, Path projectDir) throws IOException {

--- a/modules/jooby-cli/src/main/resources/cli/App.java.hbs
+++ b/modules/jooby-cli/src/main/resources/cli/App.java.hbs
@@ -1,7 +1,7 @@
 package {{package}};
 
 import io.jooby.Jooby;
-import io.jooby.netty.NettyServer;
+import io.jooby.{{serverPackageName}}.{{serverClassName}};
 {{#if openapi}}
 import io.jooby.OpenAPIModule;
 {{/if}}
@@ -9,7 +9,7 @@ import io.jooby.OpenAPIModule;
 public class App extends Jooby {
 
   {
-    install(new NettyServer());
+    install(new {{serverClassName}}());
   {{#if openapi}}
     install(new OpenAPIModule());
   {{/if}}

